examples/esp/build.rs

```diff
@@ -0,0 +1,43 @@
+fn main() {
+    // Uncomment the two lines below to enable defmt support
+    //linker_be_nice();
```

Comment:
`linker_be_nice` is good to have always enabled, it'll not generate false-positives if you don't use defmt